src/app.js

```diff
@@ -1,31 +1,35 @@
-'use strict';
-
-/* istanbul ignore next */ if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')
-  require('dotenv').config(); // eslint-disable-line import/no-extraneous-dependencies
-
-const express = require('express');
+// eslint-disable-next-line import/no-extraneous-dependencies
+import dotenv from 'dotenv';
+import express from 'express';
+import { dirname } from 'path';
```

Comment:
The node:module pattern should be everywhere and not just here and there

```suggestion
import { dirname } from 'node:path';
```

Reply:
👍 thanks.

Reply:
there's a bit more to deal with, want me to mark all of them for you?